docs/operation/Recover_From_Etcd_Permanent_Quorum_Loss.md

```diff
@@ -0,0 +1,83 @@
+
+## ETCD clusters in shoot control plane
+If permanent quorum loss occurs to a [Multinode ETCD cluster](https://github.com/gardener/etcd-druid/tree/master/docs/proposals/multi-node), the operator needs to note down the PVCs, configmaps, statefulsets, CRs etc related to that ETCD cluster and work on those resources only. Following steps guide a human operator to recover from permanent quorum loss of a ETCD cluster. We assume the name of the ETCD CR for the ETCD cluster is `etcd-main`
+
+   1. Add the following annotation to the etcd object `k annotate etcd etcd-main druid.gardener.cloud/ignore-reconciliation="true"`
+   3. Note down the configmap name that is attached to the `etcd-main` statefulset. If you describe the statefulset with `k describe sts etcd-main`, look for the lines similar to following lines to identify attached configmap name. It will be needed at later stages:
+
+   ```
+     Volumes:
+      etcd-config-file:
+       Type:      ConfigMap (a volume populated by a ConfigMap)
+       Name:      etcd-bootstrap-4785b0
+       Optional:  false
+   ```
+
+   3. Scale down the `etcd-main` statefulset replicas to 0
+
+      `kubectl scale sts etcd-main --replicas=0`
+   4. The PVCs will look like the following on listing them with the command `k get pvc` :
+      
+      ```
+      main-etcd-etcd-main-0        Bound    pv-shoot-garden-aws-c1c6154e-6506-4017-9cfc-4f1cc9afa730   80Gi       RWO            gardener.cloud-fast   385d
+      ```
+      Delete all PVCs that were attached to ETCD main cluster.
+
+      `k delete pvc -l instance=etcd-main`
+   5. Edit the `etcd-main` configmap’s (Ex: etcd-bootstrap-4785b0).
+      
+      Find the `initial-cluster` field in the configmap. It will look like the following:
+      ```
+      # Initial cluster
+        initial-cluster: etcd-main-0=https://etcd-main-0.etcd-main-peer.default.svc:2380,etcd-main-1=https://etcd-main-1.etcd-main-peer.default.svc:2380,etcd-main-2=https://etcd-main-2.etcd-main-peer.default.svc:2380
+      ```
+
+      Change the `initial-cluster` field to have only one member (etcd-main-0) in the string to make it look like the following:
+
+      ```
+      # Initial cluster
+        initial-cluster: etcd-main-0=https://etcd-main-0.etcd-main-peer.default.svc:2380
+      ```
+   6. Scale up the `etcd-main` statefulset replicas to 1
+
+      `kubectl scale sts etcd-main --replicas=1`
+   7. Wait for the single member etcd to be completely ready.
+
+      `k get pods etcd-main-0` will give the following output when ready:
+      ```
+      NAME          READY   STATUS    RESTARTS   AGE
+      etcd-main-0   2/2     Running   0          13h
```

Comment:
```suggestion
      etcd-main-0   2/2     Running   0          1m
```
When the sts is scaled up, the pod would have started up recently, rather than `13h` ago, right?